arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -286,5 +287,28 @@ public async Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>(string col
             }
         }
 
+        /// <summary>
+        /// Returns an array of all keys, ids, or URI paths for all documents in the
+        /// collection identified by collection.The type of the result array is
+        /// determined by the type attribute.
+        /// Note that the results have no defined order and thus the order should
+        /// not be relied on.
+        /// </summary>
+        /// <param name="collectionName"></param>
+        /// <param name="body"></param>
+        /// <returns></returns>
+        public async Task<AllDocumentsResponse> ReadAllDocumentsAsync(string collectionName, AllDocumentsBody body)
```

Comment:
I think we have better add the `[Obsolete]` attribute here. This is what the doc says:

> This route should no longer be used.
All endpoints for Simple Queries are deprecated from version 3.4.0 on.
They are superseded by AQL queries.

Actually should we include that at all?